data/Store.js

```diff
@@ -53,6 +53,9 @@ export class Store {
     _filter = null;
     _loadRootAsSummary = false;
 
+    /** @private -- used internally by StoreFilterField that is bound to this store. */
+    @bindable xhFilterText = null;
```

Comment:
Suggest tweak to "by any StoreFilterField bound to this store" to make it clear that there's not one bound by default somehow.